src/lib/y2storage/storage_manager.rb

```diff
@@ -163,7 +163,9 @@ def deactivate
     def probe(probe_callbacks: nil)
       probe!(probe_callbacks: probe_callbacks)
       true
-    rescue Storage::Exception, Yast::AbortException
+    rescue Storage::Exception, Yast::AbortException => e
```

Comment:
I expect libstorage-ng already logs it. If so, I would avoid to log twice.

Reply:
Ok, I didn't know. Anyway, libstorage-ng probably logs something regarding to Storage::Exception but not about Yast::AbortException. 

No strong opinion about what to do, if keep logging error or reverting that change, honestly.

Reply:
If libstorage-ng already logs, then I would also log the `Yast::AbortException` before raising it.